src/main/java/com/bitquest/bitquest/BitQuest.java

```diff
@@ -812,7 +812,7 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
                         e1.printStackTrace();
                     }
                     try {
-                        if(fromWallet != null && fromWallet.balance() >= sendAmount) {
+                        if(fromWallet != null && fromWallet.balance() >= sendAmount && sendAmount >= 20000) {
```

Comment:
It might be worth doing this as a constant a la LAND_PRICE: https://github.com/bitquest/bitquest/blob/374c95604d1dabbc05e1491e588acb1aee6c71d2/src/main/java/com/bitquest/bitquest/BitQuest.java#L83